sos/report/plugins/sssd.py

```diff
@@ -23,12 +24,16 @@ class Sssd(Plugin):
     def setup(self):
         self.add_copy_spec([
             "/etc/sssd/sssd.conf",
-            "/var/log/sssd/*",
             "/var/lib/sss/pubconf/krb5.include.d/*",
             # SSSD 1.14
             "/etc/sssd/conf.d/*.conf"
         ])
 
+        # add individual log files
+        logdir = "/var/log/sssd/"
+        for logfile in listdir(logdir):
+            self.add_copy_spec([logdir + logfile])
```

Comment:
While the concatenation of strings with path+filename should work, it is safer to have there `os.path.join(logdir,logfile)`. Anyway, we might end up in a different approach that does not need this at all.